js/main.js

```diff
@@ -1,2 +1,34 @@
 /* global data */
 /* exported data */
+
+var photoURL = document.querySelector('#photo-url');
+var img = document.querySelector('img');
+var newEntryForm = document.querySelector('form');
+
+photoURL.addEventListener('input', photoURLUpdate);
+
+function photoURLUpdate(event) {
+  img.setAttribute('src', event.target.value);
+  if (img.getAttribute('src') === '') {
+    img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  }
+}
+
+newEntryForm.addEventListener('submit', submitForm);
+
+function submitForm(event) {
+  event.preventDefault();
+  var inputValues = {};
+  var titleValue = newEntryForm.elements.title.value;
+  var urlValue = newEntryForm.elements['photo-url'].value;
+  var notesvalue = newEntryForm.elements.notes.value;
+  inputValues.title = titleValue;
+  inputValues.url = urlValue;
+  inputValues.notes = notesvalue;
+  inputValues.nextEntryID = data.nextEntryID;
```

Comment:
You should delete this line. The line below this is the proper way to save an id on the entry, the `nextEntryId` property name makes no sense for an individual entry.